oauth2-server/src/main/java/com/clouway/oauth2/TokenInfoController.java

```diff
@@ -31,9 +31,12 @@ public Response handleAsOf(Request request, DateTime instantTime) {
     BearerToken token = possibleToken.get();
 
     JsonObject o = new JsonObject();
+    o.addProperty("azp", token.clientId);
```

Comment:
It would be better IdentityFinder to be taken in the TokenInfoController instead of Tokens implementation is it won't force change on clients.